src/averageAges.js

```diff
@@ -37,7 +49,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(woman => woman.sex === 'f');
+
+  if (withChildren) {
+    women = women.filter(woman =>
+      people.find(person => person.mother === woman.name));
+  }
```

Comment:
```suggestion
  let women = people.filter(woman => withChildren 
  ? people.find(person => person.mother === woman.name)) 
  : woman.sex === 'f');

```
if заборонені в цьому завданні